Sources/iosDesignTokens/todaytix/GradientTokens.swift

```diff
@@ -0,0 +1,149 @@
+
+// Do not edit directly
+// Generated on Thu, 10 Nov 2022 19:19:40 GMT
+
+
+import UIKit
+
+public class GradientTokens {
+	public var lightInteractiveInteractive01: CAGradientLayer {
+		get {
+			let gradientLayer = CAGradientLayer()
+			gradientLayer.colors = [UIColor(red: 0.871, green: 0.149, blue: 0.318, alpha:1), UIColor(red: 0.882, green: 0.145, blue: 0.157, alpha:1)]
+			gradientLayer.startPoint = CGPoint(x: 0.5, y: 0.05)
+			gradientLayer.endPoint = CGPoint(x: 0.5, y: 0.95)
+			return gradientLayer
+		}
+	}
+
+	public var lightTagBrandBackground: CAGradientLayer {
+		get {
+			let gradientLayer = CAGradientLayer()
+			gradientLayer.colors = [UIColor(red: 0.871, green: 0.149, blue: 0.318, alpha:1), UIColor(red: 0.882, green: 0.145, blue: 0.157, alpha:1)]
+			gradientLayer.startPoint = CGPoint(x: 0.5, y: 0.05)
+			gradientLayer.endPoint = CGPoint(x: 0.5, y: 0.95)
+			return gradientLayer
+		}
+	}
+
+	public var lightTagBrandHover: CAGradientLayer {
+		get {
+			let gradientLayer = CAGradientLayer()
+			gradientLayer.colors = [UIColor(red: 1.000, green: 0.059, blue: 0.341, alpha:1), UIColor(red: 1.000, green: 0.071, blue: 0.149, alpha:1)]
+			gradientLayer.startPoint = CGPoint(x: 0.5, y: 0.05)
+			gradientLayer.endPoint = CGPoint(x: 0.5, y: 0.95)
+			return gradientLayer
+		}
+	}
+
+	public var coreGradientTangerine: CAGradientLayer {
+		get {
+			let gradientLayer = CAGradientLayer()
+			gradientLayer.colors = [UIColor(red: 0.765, green: 0.137, blue: 0.137, alpha:1), UIColor(red: 1.000, green: 0.506, blue: 0.333, alpha:1)]
+			gradientLayer.startPoint = CGPoint(x: 0.5, y: 0.05)
+			gradientLayer.endPoint = CGPoint(x: 0.5, y: 0.95)
+			return gradientLayer
+		}
+	}
+
+	public var coreGradientSky: CAGradientLayer {
+		get {
+			let gradientLayer = CAGradientLayer()
+			gradientLayer.colors = [UIColor(red: 0.000, green: 0.208, blue: 0.463, alpha:1), UIColor(red: 0.114, green: 0.584, blue: 0.851, alpha:1)]
+			gradientLayer.startPoint = CGPoint(x: 0.5, y: 0.05)
+			gradientLayer.endPoint = CGPoint(x: 0.5, y: 0.95)
+			return gradientLayer
+		}
+	}
+
+	public var coreGradientRaspberry: CAGradientLayer {
+		get {
+			let gradientLayer = CAGradientLayer()
+			gradientLayer.colors = [UIColor(red: 0.502, green: 0.024, blue: 0.220, alpha:1), UIColor(red: 0.882, green: 0.090, blue: 0.408, alpha:1)]
```

Comment:
are these all just auto generated? I feel like we don't have these anywhere

Reply:
@flitterio Isn't this the gradients used for `get tickets` buttons and similar?

Reply:
yeah, we use gradients for the primary action button and I needed to see if gradients could work in this format